src/plugins/charts/public/static/components/color_schema.tsx

```diff
@@ -70,7 +70,7 @@ function ColorSchemaOptions({
   }, [uiState]);
 
   const resetColorsButton = (
-    <EuiText size="xs">
+    <EuiText size="s">
```

Comment:
Can you point of this screenshot?

Reply:
![image](https://github.com/opensearch-project/OpenSearch-Dashboards/assets/32652829/b8a9c913-09fe-4ad8-8b9e-c755b30f0f8a)
It's a link on heat map visualization , but as suggested to keep consistency with other link, for this one we can remain xs(12px)

Reply:
Yeah, that makes sense